third_party/forked/shell2junit/sh2ju.sh

```diff
@@ -32,6 +32,15 @@ juDIR=`pwd`/results
 # The name of the suite is calculated based in your script name
 suite=""
 
+if LANG=C sed --help 2>&1 | grep -q GNU; then
+  SED="sed"
+elif which gsed &>/dev/null; then
+  SED="gsed"
+else
+  echo "Failed to find GNU sed as sed or gsed. If you are on Mac: brew install gnu-sed." >&2
```

Comment:
This seems like a big ask for every dev on Mac -- are we ready to do that? Did we ask mailing lists?

Reply:
We already do this in a number of scripts using this function: https://github.com/kubernetes/kubernetes/blob/884e4b7b9c63a9a3b94278972d640454b8d16e47/hack/lib/util.sh#L779-L794

I've just copy pasted it over into the forked junit code, because I was getting errors with the SEDs in it on my Mac.

Reply:
ACK